community/front-end/ofe/deploy.sh

```diff
@@ -105,6 +105,26 @@ HELP1
         (or Service Account Admin)
       - Project IAM Admin
 
+  Usage: ./deploy.sh [--config <path-to-config-file>] [--help]
+
+      --config <path-to-config-file> : path to YAML configuration file containing deployment variables.
+                                       If not specified, script will prompt user for input.
+      --help                         : display this help message
+
+  If --config option is used, all variables required for deployment must be specified in the YAML
+  file. The script will not prompt for any input in this case.
+
+  Example YAML file:
+  deployment_name: MyDeployment
+  project_id: my-project-id
+  zone: us-west1-a
+  subnet_name: my-subnet (optional)
+  dns_hostname: myhostname.com (optiona)
+  ip_address: 1.2.3.4 (optional)
+  django_superuser_username: sysadm
+  django_superuser_password: Passw0rd!
```

Comment:
Writing a password in plain text here is probably not the best, is there any other mechanism we can think about?

Reply:
One way or another, you will need to provide the password to the deployment script. We can have another argument such as `--django-password` or use environment variable. But I think you would still expose the password in the plain text anyway, just in a different place. I am open for discussion, let me know of any ideas.

Reply:
This is better, thanks. That allows the env variable to be set from a cloud secrets manager and then fed into cloud build, for instance.